### DISCUSSION TITLE
How can I add static locale files?

### DISCUSSION THREAD

**Discussion by YunusEmreAlps** [requarks/wiki#5718] [Help / Questions] [answered]
Hello,

I hope you're well and everything is good for you.

I added new features like in-page search, export files, etc... to the wiki.js project. I need to add new strings for the platform. How can I do that? 

I want to localization to the platform with static files.

Thank you very much.
Best regards,

**Comment by NGPixel** [comment, marked as answer]
https://docs.requarks.io/dev

**Reply by YunusEmreAlps** [reply]
No, I want to customize my own locale files. When I add my own locale files just works for developer mode.

**Reply by NGPixel** [reply]
Again, read the docs I linked. You need to build the client assets which can then be used in a production installation.
